tests/build.rs

```diff
@@ -139,21 +143,8 @@ fn it_fails_with_multiple_webpack_configs() {
     utils::create_temporary_copy(fixture);
 
     single_env_settings! {fixture, r#"
-        type = "Webpack"
-    "#};
-
-    build_fails_with(fixture, "Multiple webpack configurations are not supported. You can specify a different path for your webpack configuration file in wrangler.toml with the `webpack_config` field");
-    utils::cleanup(fixture);
-}
-
-#[test]
-fn it_fails_with_multiple_specify_webpack_configs() {
```

Comment:
should this test still exist? it's kind of redundant but i don't think its outcome is changed by this PR.

Reply:
It's the same test as `it_fails_with_multiple_webpack_configs` the only difference previously was one had inferred `webpack.config.js` existed and one had it in the toml - after this breaking change they are all "specify" tests